src/main/java/ch/wisv/events/core/model/registration/Study.java

```diff
@@ -0,0 +1,50 @@
+package ch.wisv.events.core.model.registration;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Setter;
+import org.hibernate.validator.constraints.NotEmpty;
+
+/**
+ * Study data object.
+ */
+@Entity
+@Data
+public class Study {
+
+    /**
+     * Field id of the study.
+     */
+    @Id
+    @GeneratedValue
+    @Setter(AccessLevel.NONE)
+    private Integer id;
+
+    /**
+     * Study name.
+     */
+    @NotEmpty
+    private String studyName;
```

Comment:
Just three choices here, have you thought about reflecting this in the back-end?

Reply:
Changed it to an enum.